vtStorAtaProtocol/Platform/Linux/vtStorAtaProtocolPlatformDefines.h

```diff
@@ -24,4 +24,6 @@ limitations under the License.
 #define VT_STOR_ATA_PROTOCOL_API
 #endif
 
+#define IOCTL_SG_IO_ERROR -1

```

Comment:
I discussed with a.Dien. I defined it in ProtocolAtaPassThrough.h, but i think need move it to define for Platform.